languages/tree-sitter-stack-graphs-java/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+name = "tree-sitter-stack-graphs-java"
+version = "0.1.0"
+description = "Stack graphs for precise code navigation for the Java programming language"
```

Super nit: This repo is not specific to GH precise code navigation product.
```suggestion
description = "Stack graphs for the Java programming language"
```